style.css

```diff
@@ -0,0 +1,179 @@
+@import url("https://fonts.googleapis.com/css2?family=Edu+TAS+Beginner:wght@400;500;600;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Edu+TAS+Beginner:wght@400;500;600;700&family=Lobster&family=Rubik+Moonrocks&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Edu+TAS+Beginner:wght@400;500;600;700&family=Rubik+Moonrocks&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.header__tasks_delete img {
+  width: 3vw;
+  height: 5vh;
+  background-size: cover;
+  margin: 2%;
+  padding: 1%;
+  box-shadow: 0 0 5px 1px #34142d;
+  border-radius: 20%;
+  backdrop-filter: blur(5px);

```

Comment:
Why do you use so many different values: %, px, vh? Use one, maximum two and then as an exception

Reply:
didn't fix

Reply:
Why are percents and pixels left?